app/controllers/submit.js

```diff
@@ -630,24 +628,19 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
                     label: 'Submit - Save and Continue, Existing Node Existing File'
                 });
 
-            const node = this.get('node');
-            const currentTitle = node.get('title');
+            const model = this.get('model');
+            const currentTitle = model.get('title');
             const nodeTitle = this.get('nodeTitle');
 
             this.set('basicsAbstract', this.get('node.description') || null);
```

Comment:
Should this be `model.description` here?

Reply:
In case where user adds a file from an existing node, its description `node.description` is used as placeholder text for the abstract in the basics `basicsAbstract` section. While this can be changed to use `model.description`, I thought we can keep as `node.description` till further in the divorce when we know if this feature will be supported.